### ISSUE TITLE
update sytem stf project name

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/aqa-tests#2949]
System stf git repo name/project has changed to `STF`,  to make it work on case-sensitive platform we need to update the name accordingly. Otherwise we will get the following error 

```/home/runner/work/run-aqa/run-aqa/aqa-tests/system/common.xml:194: The following error occurred while executing this line:
/home/runner/work/run-aqa/run-aqa/aqa-tests/system/common.xml:60: The directory /home/runner/work/run-aqa/run-aqa/aqa-tests/system/stf does not exist
```

For example it happened run-aqa github action on linux
https://github.com/adoptium/run-aqa/runs/3796240585?check_suite_focus=true

To fix it, the `stf` in https://github.com/adoptium/aqa-tests/blob/master/system/common.xml should be replaced by `STF`

**Comment by sophia-guo** [comment]
Looks like https://github.com/adoptium/STF may need to update. I will take a look and if the change should be there I will transfer this PR to STF repo.

**Comment by abigailsleek** [comment]
I will like to assign this to myself.

**Comment by Lovelynchristian** [comment]
I would love this to be assigned to me.

**Comment by keithc-ca** [comment]
It might be better to go the other way: update scripts to use a working directory based on the actual repo name (`STF` instead of `stf`).

**Comment by llxia** [comment]
@Lovelynchristian do you have any questions regarding this issue?

**Comment by llxia** [comment]
Thanks @xius666

**Comment by sophia-guo** [comment]
Agreed https://github.com/adoptium/aqa-tests/issues/2949#issuecomment-945911127. There are multiple places in aqa-tests and aqa-systemtest are using as `directoryVariable/stf/....`. Setting a working directory based on the actual repo name (STF instead of stf) should make the job easier. For now we'd like this first be addressed.

Probably open a separate issue to improve this(Setting a working directory based on the actual repo name for STF. @Mesbah-Alam thoughts?
